### PR TITLE
docs: nx create stacks workspace plugin docs

### PR DESCRIPTION
**Pull request by KarimHmaissi**

### CODE REVIEW

File: docs/nx/create-stacks-workspace/plugin-information.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Create a new Stacks Nx workspace. Using the create-stacks-workspace script will quickly allow you to initialize a new NX workspace with the Stacks recommended plugins, packages and configuration to get started on a new project quickly.
+
+The Stacks `create-stacks-workspace` script will 
+- Run Nx [create-nx-workspace](https://nx.dev/packages/nx/documents/create-nx-workspace) command to setup a new NX workspace passing along any command line arguments provided 
+- Install any recommended plugin/generators/executors from Stacks (the exact plugins installed will depend on the preset selected see [presets](/docs/nx/create-stacks-workspace/ensono-stacks-create-stacks-workspace#presets))
+- Configure the NX workspace 
+- Execute recommended generators from Stacks to initialize the workspace/projects depending on the options provided.
```
Comment (Danphillipz) [inline docs/nx/create-stacks-workspace/plugin-information.md:24]: Could we expand? Then link to which generators will be ran by default depending on your preset?

File: sidebars.js [hunk 2]
```diff
@@ -281,6 +281,7 @@ module.exports = {
         type: "category",
         label: "Stacks Plugins",
         items: [
+          "nx/create-stacks-workspace/ensono-stacks-create-stacks-workspace",
```
Comment (Danphillipz) [inline sidebars.js:284]: Could we also update the main /docs/nx/nx_stacks#option-2-create-stacks-workspace-generator section to point to this page?

**Review by Danphillipz — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 2 above)
Comment (KarimHmaissi) [reply sidebars.js:284]: Added link

File: docs/nx/create-stacks-workspace/plugin-information.md (hunk 1 above)
Comment (KarimHmaissi) [reply docs/nx/create-stacks-workspace/plugin-information.md:24]: I believe as it stands the only ensono plugin that is run is the Next plugin. With the empty preset and the react preset no ensono plugins are run. This does not include the workspace plugin which will always run. Will add this to documentation

**Review by Danphillipz — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**